src/bin/lpython.cpp

```diff
@@ -859,6 +1005,7 @@ int main(int argc, char *argv[])
         // LSP specific options
         app.add_flag("--show-errors", show_errors, "Show errors when LSP is running in the background");
         app.add_flag("--show-document-symbols", show_document_symbols, "Show symbols in lpython file");
+        app.add_option("--goto-def", goto_def_index, "Pass index for GoTo Definition");
```

Comment:
VSCode calls it `Go to Definition`, so maybe we can call this option `--go-to-definition`?

Reply:
Also, it seems the input almost always is a line and column number? If so, that should be the input here as well. So the description would be "Return a definition for the symbol at the given line and column", and if this is too long, we can shorten it.